stress/basic.go

```diff
@@ -678,8 +678,11 @@ func (o *outputConfig) HTTPHandler(method string) func(r <-chan response, rt *Ti
 			Precision:       "ns",
 		})
 		for p := range r {
+			tags := make(map[string]string)
```

Comment:
The reason why I didn't do that is because then we would likely want to take a lock on the tags so that `len(o.tags)` doesn't have a race condition. I didn't want to take a lock while allocating memory when I don't think it will have a substantial difference in performance as the size of the map is so small.